docs/GettingStartedDocs/Contributors/ExperimentalSupportRHEL8.md

```diff
@@ -0,0 +1,99 @@
+# Experimental - Getting Started with Open Enclave on RHEL8
+
+## Platform requirements
+
+- RHEL8
+- A system which is SGX1 or SGX1+FLC capable. Please refer to [SGXSupportLevel.md](/docs/GettingStartedDocs/SGXSupportLevel.md) to determine the SGX support level for your target system.
+
+Note: Attestation is not supported on RHEL8.
+
+## Clone Open Enclave SDK repo from GitHub
+
+Use the following command to download the source code (make sure `git` is installed before doing this):
+
+```bash
+git clone https://github.com/openenclave/openenclave.git
+```
+
+This creates a source tree under the directory called openenclave.
+
+## Install project requirements
+
+First, change directory into the openenclave repository:
+
+```bash
+cd openenclave
+```
+
+Run the command below to install prerequisites.
+
+```bash
+ansible-playbook scripts/ansible/oe-contributors-setup-sgx1.yml
+```
+
+## Build
+
+To build, first create a build directory ("build/" in the example below) and change into it.
+
+```bash
+mkdir build/
+cd build/
+```
+
+Then run `cmake` to configure the build and generate the make files and build:
+
+```bash
+export CC=gcc
```

Comment:
Why are the exports needed? Is there an issue with the default CC and CXX?

Reply:
Verified CC and CXX are not defined on RHEL.

Reply:
@anakrish Clang is a default compiler on OE, but there are some errors on Redhat while building OE. Sorry I have not figured out the fix.